### PR TITLE
Fixed some typos for pkg/proxy

### PR DESCRIPTION
**Pull request by hw-qiaolei**
Fixed some typos for `pkg/proxy` and `pkg/util/iptables`.

### CODE REVIEW

**Comment by hw-qiaolei** [comment]
@eparis thanks for reviewing this. Pls remove the 'ok-to-merge' label, let me squash these commits to one commit.

File: pkg/util/iptables/iptables.go [hunk 1]
```diff
@@ -382,7 +382,7 @@ func (runner *runner) checkRule(table Table, chain Chain, args ...string) (bool,
 
 // Executes the rule check without using the "-C" flag, instead parsing iptables-save.
 // Present for compatibility with <1.4.11 versions of iptables.  This is full
-// of hack and half-measures.  We should nix this ASAP.
```
Comment (thockin) [inline pkg/util/iptables/iptables.go]: this one is correct as-is:  nix means "get rid of"

Comment (hw-qiaolei) [inline pkg/util/iptables/iptables.go]: @thockin thanks for pointing this. I will revert it.

**Comment by hw-qiaolei** [comment]
@thockin squash commits and revert 'nix' is done, pls check, thanks.

I learned another word 'nix' which means 'get rid of' from you, many thanks for this, too :)

**Pull request merged**